fee.go

```diff
@@ -81,6 +81,7 @@ type FeeItem struct {
 	Code       							string      `json:"code,omitempty"`
 	Name      							string      `json:"name,omitempty"`
 	InvoiceDisplayName			string      `json:"invoice_display_name,omitempty"`
+	GroupInvoiceDisplayName	string      `json:"group_invoice_display_name,omitempty"`
```

Comment:
it looks like the indentation is a bit weird compare to the other attributes. Could you align those two `DisplayName`?

Reply:
Fixed.